product_docs/docs/biganimal/release/overview/02_high_availability.mdx

```diff
@@ -4,25 +4,44 @@ redirects:
   - 02_high_availibility
 ---
 
-BigAnimal enables deploying a cluster with or without high availability. The option is controlled with the **High Availablity** control on the [Create Cluster](https://portal.biganimal.com/create-cluster) page in the [BigAnimal](https://portal.biganimal.com) portal.    
+BigAnimal supports three deployment architectures: 
+- Single node
+- Standard high availability
+- Extreme high availability (beta)
 
-## High availability—enabled
+You choose which type of deployment you want when creating your cluster on the [Create Cluster](https://portal.biganimal.com/create-cluster) page in the [BigAnimal](https://portal.biganimal.com) portal.  
```

Comment:
Shouldn't this say "type of cluster", not "type of deployment"?

Reply:
It should say "type of cluster" to match the UI